### ISSUE TITLE
There is no 404 page.

### ISSUE THREAD

**Issue by bytesagar** [OWASP/NodeGoat#201]
## Added 404 page, if the page is not found
* PR #200

**Comment by UlisesGascon** [comment]
Hi @sagar608!

Thanks for open this issue and fix the 404 missing page in a PR.

We are working now in the Lerna implementation #187. Once the PR #189 is merged the project setup will change in a new folder structure. 

Can you create a PR for the 404 based on branch `feature/187`? The CI should work fine and prevent linting issues.

**Comment by sagar608** [comment]
Thank you @UlisesGascon 
If you could help me on how can I create a new PR on branch  > feature/187 ?

**Comment by lirantal** [comment]
Let's take it with the created PR
